### DISCUSSION TITLE
Hitobjects in Lazer fade differently then stable

### DISCUSSION THREAD

**Discussion by Tunnelbliick** [ppy/osu#20801] [Q&A] [answered]
### Type
Gameplay

### Description
This all started when i made a comparison video between lazer and stable gameplay and noticed some fading differences when i took a close look.

Ive noticed that Hitobjects (Sliders) in Lazer fade more "linear" then in stable.
Best noticed on .25x speed looking at slider fade out.

Also noticed that Spinners dont fade in or out properly.

Lazer version - 2022.1008.2-lazer
Stable version - b20221016.2

### Video
[Video here](https://www.youtube.com/watch?v=b332I_thiLg) (this one was on cutting edge rest is on stable)
[2nd auto video](https://www.youtube.com/watch?v=OG5Op0XKnHA) (100% synced / still processing thanks yt)

Sliders wrong fade out .10x speed @ 120 fps
[Video here](https://youtu.be/rAa1OU3BLsw)

Spinner wrong fade in .25x speed @ 120 fps  **(Lazer is 23ms later then stable)**
[Video here](https://www.youtube.com/watch?v=98aINVow9CQ)

Spinner wrong fade out .10x speed @ 120 fps **(Lazer is 9ms faster then stable)**
[Video here](https://www.youtube.com/watch?v=5Njr3d2SHkk)

### Screenshots
Slider fade out
![Slider Fade out](https://user-images.githubusercontent.com/38663241/196214969-f9c24d9d-1ed2-4970-b9ee-5f16fbefcd52.png)
Spinners not fading corretly
![Spinners not Fading in](https://user-images.githubusercontent.com/38663241/196214904-f2af1a6a-a6c9-4361-a5ae-505b1b735112.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20806
